### PR TITLE
test menu,findElement,login

### PR DESCRIPTION
**Pull request by Jannatel**

### CODE REVIEW

File: src/main/java/base/CommonApi.java [hunk 1]
```diff
@@ -22,23 +19,23 @@ public class CommonApi {
...
-                System.setProperty("webdriver.chrome.driver", currentDir+"\\driver\\mac\\chromedriver");
+                System.setProperty("webdriver.chrome.driver","/home/free/IdeaProjects/BlackList_FrameWork/driver/linux/chromedriver");
                 driver = new ChromeDriver();
             }else if (browser.equalsIgnoreCase("firefox")){
                 System.setProperty("webdriver.gecko.driver", currentDir+"\\driver\\mac\\geckodriver");
                 driver = new FirefoxDriver();
             }
         }else if (os.equalsIgnoreCase("linux")){
             if (browser.equalsIgnoreCase("chrome")){
-                System.setProperty("webdriver.chrome.driver", currentDir+"\\driver\\linux\\chromedriver");
```
Comment (SalimHebib) [inline src/main/java/base/CommonApi.java:41]: approved

File: src/main/java/base/CommonApi.java [hunk 2]
```diff
@@ -22,23 +19,23 @@ public class CommonApi {
...
+                System.setProperty("webdriver.chrome.driver","/home/free/IdeaProjects/BlackList_FrameWork/driver/linux/chromedriver");
                 driver = new ChromeDriver();
             }else if (browser.equalsIgnoreCase("firefox")){
                 System.setProperty("webdriver.gecko.driver", currentDir+"\\driver\\windows\\geckodriver.exe");
                 driver = new FirefoxDriver();
             }
         }else if (os.equalsIgnoreCase("mac")){
             if (browser.equalsIgnoreCase("chrome")){
-                System.setProperty("webdriver.chrome.driver", currentDir+"\\driver\\mac\\chromedriver");
+                System.setProperty("webdriver.chrome.driver","/home/free/IdeaProjects/BlackList_FrameWork/driver/linux/chromedriver");
```
Comment (SalimHebib) [inline src/main/java/base/CommonApi.java:30]: approved

File: src/main/java/base/CommonApi.java [hunk 3]
```diff
@@ -22,23 +19,23 @@ public class CommonApi {
     public void getDriver(String browser, String os){
         if (os.equalsIgnoreCase("windows")){
             if (browser.equalsIgnoreCase("chrome")){
-                System.setProperty("webdriver.chrome.driver", currentDir+"\\driver\\windows\\chromedriver.exe");
```
Comment (SalimHebib) [inline src/main/java/base/CommonApi.java:25]: approved

File: src/main/java/base/CommonApi.java [hunk 4]
```diff
@@ -1,9 +1,6 @@
 package base;
 
-import org.openqa.selenium.By;
-import org.openqa.selenium.Keys;
```
Comment (SalimHebib) [inline src/main/java/base/CommonApi.java:4]: did u make change over here ?

File: src/test/resources/walgreens-runner.xml [hunk 5]
```diff
@@ -5,9 +5,13 @@
     <parameter name="url" value="https://www.walgreens.com/"/>
     <parameter name="browser" value="chrome"/>
     <parameter name="os" value="linux"/>
-    <test name="">
```
Comment (SalimHebib) [inline src/test/resources/walgreens-runner.xml:8]: approved

File: src/test/resources/walgreens-runner.xml [hunk 6]
```diff
@@ -5,9 +5,13 @@
...
+    <test name="wlgreens">
         <classes>
-            <class name=""/>
+<!--        <class name="walgreens.elementFinder.SearchElement"/>-->
```
Comment (SalimHebib) [inline src/test/resources/walgreens-runner.xml:10]: approved

File: src/main/java/base/CommonApi.java [hunk 7]
```diff
@@ -1,9 +1,6 @@
...
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
```
Comment (SalimHebib) [inline src/main/java/base/CommonApi.java:6]: this one ?

**Review by SalimHebib — APPROVED** [review]

**Review by SalimHebib — APPROVED** [review]

**Review by SalimHebib — APPROVED** [review]

**Review by SalimHebib — APPROVED** [review]

**Review by SalimHebib — APPROVED** [review]

**Review by SalimHebib — APPROVED** [review]

**Comment by Jannatel** [comment]
hi

**Pull request closed without merging**